eclair-core/src/test/scala/fr/acinq/eclair/blockchain/bitcoind/BitcoindService.scala

```diff
@@ -66,6 +72,28 @@ trait BitcoindService extends Logging {
   var bitcoinrpcclient: BitcoinJsonRPCClient = _
   var bitcoinrpcauthmethod: BitcoinJsonRPCAuthMethod = _
   var bitcoincli: ActorRef = _
+  val onchainKeyManager = new LocalOnchainKeyManager(ByteVector.fromValidHex("01" * 32), Block.RegtestGenesisBlock.hash)
+
+  def setExternalSignerScript(keyManager: OnchainKeyManager): Unit = {
```

Comment:
Yes we need to instantiate different fresh HWI-based bitcoin wallets in some of the tests.

Reply:
After we have created the watch-only descriptor wallet in `waitForBitcoindReady` couldn't we just use the `importdescriptors` RPC call instead of defining a `signer` script in `startBitcoind`?

Reply:
I created a test script [start_bitcoin_descriptor.sh](https://github.com/remyers/lightning-cfg/blob/descriptor-wallet/scripts/start_bitcoin_descriptor.sh) that shows how I would expect to initialize the descriptor wallet for testing without using an external signing script.

Reply:
Great job, fixed it in [53c3c77](https://github.com/ACINQ/eclair/pull/2613/commits/53c3c7733ac8d3ff88953973f9f9ecbbfcc99e40). Conceptually more clear now.